src/main/io/gps.c

```diff
@@ -281,6 +281,10 @@ void gpsInit(void)
 
     gpsData.lastMessage = millis();
 
+    if (gpsConfig()->provider == GPS_MSP) { // no serial ports used when GPS_MSP is configured
+        return;
```

Comment:
I think we need to set `gpsSetState(GPS_RECEIVING_DATA)` here as well.

Reply:
Hmmmm...
During startup it will initialize into ```GPS_RECEIVING_DATA```.
Then it most probably will run into a timeout and switch to ```GPS_LOST_COMMUNICATION```.
Then it will start receiving data all of a sudden because the MSP device is ready and switch back to ```GPS_RECEIVING_DATA```.
If we leave it at ```GPS_UNKNOWN``` nothing happens until data is actually received.

What benefit do you see with this?

Reply:
Maintainability. Currently, there is no transition from `GPS_UNKNOWN` to `GPS_RECEIVING_DATA`, and it won't be obvious to somebody working on the code that there is one after this change. Maybe we should introduce a new status `GPS_INITIALISED`.

Reply:
fixed